src/App.js

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+
+import { Component } from "react";
 import "./App.css";
 import ContactList from "./ContactList";
 import Feed from "./Feed";
+import Profile from "./Profile";
 
-function App() {
+class App extends Component () {
```

Comment:
```suggestion
class App extends Component {
```
You shouldn't put parentheses after `Component`. Remember, this is a **class** component, _not_ a **functional** component, so you don't need to put parentheses as if it's a function!